flask_turbo_boost/project/application/utils/flask_dance.py

```diff
@@ -1,7 +1,7 @@
 from flask import flash
 from application.models import db, OAuth, User
 from flask_dance.contrib.facebook import make_facebook_blueprint, facebook
-from flask_dance.consumer.backend.sqla import SQLAlchemyBackend
+from flask_dance.consumer.storage.sqla import SQLAlchemyBackend
```

Comment:
should we also rename `SQLAlchemyBackend` to `SQLAlchemyStorage`

ref: https://github.com/singingwolfboy/flask-dance#storages

Reply:
Ah, good catch. I've pushed another commit